lago/prefix.py

```diff
@@ -1225,6 +1226,7 @@ def _deploy_host(self, host):
                 return
 
             with LogTask('Wait for ssh connectivity'):
+                time.sleep(5)
```

Comment:
In practice, no VM loads in less than 5 seconds, so why bother it with useless SSH requests?
You should not put it inside the wait_for_ssh(), as this wait can be used elsewhere. It's only on the very first call, when the VM loads.